common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayWorkImageInclude.scala

```diff
@@ -0,0 +1,19 @@
+package uk.ac.wellcome.display.models
+
+import io.circe.generic.extras.JsonKey
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.ac.wellcome.models.work.internal.{Identified, Minted, UnmergedImage}
+
+@Schema(
+  name = "Image",
+  description = "A partial Image included on a work"
+)
+case class DisplayWorkImageInclude(
+  @Schema(description = "The image ID") id: String,
+  @JsonKey("type") @Schema(name = "type") ontologyType: String = "Image"
+)
```

Comment:
We could potentially use the original `DisplayImage` model for this, by making the other fields optional. This would have the advantage of being more DRY, and easier to introduce more data to the include if required.

Reply:
I don't think we can rely on wanting an exact subset of that model though, and those fields aren't optional when we get them from the images endpoint - think my preference is to keep it separate while we don't know what direction it's going to take rather than touching existing code

Reply:
Cool. It was mainly the word "partial" it the description which indicated that to me, maybe worth changing the wording in that case?

Reply:
Ah good point, will do